index.html

```diff
@@ -3,318 +3,308 @@
 
```

Comment:
Hello, why do we have so many changes? Did you format the code and did the changes? What are the lines with real changes?

Next time do 2 commits, the first with formatting and the second with real changes, ok?

Reply:
I think my IDE formatted the code... ok